bundle/manifests/cc-operator.clusterserviceversion.yaml

```diff
@@ -12,21 +12,16 @@ metadata:
             "namespace": "confidential-containers-system"
           },
           "spec": {
+            "ccNodeSelector": {
+              "matchLabels": {
+                "node-role.kubernetes.io/worker": ""
+              }
+            },
             "config": {
               "cleanupCmd": [
                 "/opt/kata-artifacts/scripts/kata-deploy.sh",
                 "reset"
               ],
-              "cleanupNodeSelector": [
```

Comment:
this is deleted because it's not in use any more?

Reply:
looks like we have it hard coded in a few places. I guess that's okay, we'll fix it in a follow up

Reply:
Hmm, `make bundle` synced with the manifests under `config`. Let me just cross-check once to be sure. Thanks for catching this.

Reply:
@jensfr as you rightly mentioned the `cleanupNodeSelector` is no longer used and hence removed from bundle.